test/e2e/playwright/transfer-orders.spec.ts

```diff
@@ -13,26 +13,118 @@ interface Order {
 interface OrdersResponse {
   body: Array<Order>;
 }
-
 test.describe('Transfer Orders', () => {
   let orderResponseBody: Array<Order>;
 
   test.beforeEach(async ({ page }) => {
     // Navigate to Data Verification and capture network responses
     const orderResponsePromise = page.waitForResponse(
       async (response) => response.url().includes('api/order') && response.ok(),
-      { timeout: 30000 },
+      { timeout: 35000 },
     );
 
     await page.goto('/data-verification');
-    expect(page.getByRole('heading', { name: 'Data Verification' })).toBeVisible();
+    await expect(page.getByTestId('accordion-group')).toBeVisible();
```

Comment:
Instead of getting the heading, it made more sense to wait for the accordion group to render